pkg/api/aim/query/query.go

```diff
@@ -344,25 +353,23 @@ func (pq *parsedQuery) parseCompare(node *ast.Compare) (any, error) {
 			case clause.Column:
 				switch op {
 				case ast.In:
+					// for `IN` statement, left parameter has to be always `string`.
+					if _, ok := left.(string); !ok {
+						return nil, eris.New("left parameter has to be a string")
```

Comment:
why do you use `eris.New` instead of the stdlib `errors.New` here?

Reply:
can you change all the `errors.New` and `fmt.Errorf` of this file to use `eris` then?

Reply:
ok, I reverted to use `errors` instead. lets change this when we will have good unit tests for such important module.